applications/app/views/survey/emailDigest404.scala.html

```diff
@@ -0,0 +1,45 @@
+@(surveyPage: Page)(implicit request: RequestHeader)
+
+@import model.Page
+@import conf.Configuration.Survey.formStackAccountName
```

Comment:
Not a blocker, but is it possible to include the `formStackAccountName` and some of the other form stack related endpoints as part of the model, rather than injecting the config directly into the view, and concatenating strings in the view. May require creating another Model type for `SurveyPage`.

eg: `https://@{formStackAccountName}.formstack.com/forms/js.php/@formName`, will be slightly cleaner if provided by the model